crypto/err/openssl.txt

```diff
@@ -598,6 +598,11 @@ EC_F_PKEY_EC_DERIVE:217:pkey_ec_derive
 EC_F_PKEY_EC_KEYGEN:199:pkey_ec_keygen
 EC_F_PKEY_EC_PARAMGEN:219:pkey_ec_paramgen
 EC_F_PKEY_EC_SIGN:218:pkey_ec_sign
+EC_F_PKEY_SM2_CTRL:274:pkey_sm2_ctrl
+EC_F_PKEY_SM2_CTRL_STR:275:pkey_sm2_ctrl_str
+EC_F_PKEY_SM2_KEYGEN:276:pkey_sm2_keygen
+EC_F_PKEY_SM2_PARAMGEN:277:pkey_sm2_paramgen
+EC_F_PKEY_SM2_SIGN:278:pkey_sm2_sign
```

Comment:
These look manually inserted.  Please take them away and let `make update` do the job instead.

Reply:
Besides, there's something off with these names, more on that in the code that defines them